trades/create_channel_sale.js

```diff
@@ -210,19 +233,22 @@ module.exports = ({action, ask, balance, lnd, logger}, cbk) => {
         },
         cbk) =>
       {
+        const tokens = !!hasFiat(askForRate.rate) ? askForRate.rate : asNumber(saleCost);
+
         return serviceOpenTrade({
```

Comment:
This method would now take a `price`, which would be the price expression, to be evaluated at the finalize step. Tokens could be left alone and then the caller would either pass price or tokens, not both.